unique_toolkit/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), 
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.26] - 2024-10-16
+- Bump `unique_sdq` version
```

Comment:
its unique_sdk with a k at the end

Reply:
Ah 😆 I've been making this same error lately.

Reply:
It's because of Sadique.